backend/test/authorizer/test_authorizer.py

```diff
@@ -2240,10 +2213,10 @@ def test_manage_project_sagemaker_resource_boto_error(
         "ResponseMetadata": {"HTTPStatusCode": 400},
     }
     mock_project_dao.get.return_value = MOCK_PROJECT
-    if path_param_key in ["notebookName", "endpointName", "modelName", "endpointConfigName"]:
+    if path_param_key in ["notebookName", "endpointName", "modelName", "endpointConfigName", "clusterId"]:
         mock_resource_metadata_dao.get.side_effect = ClientError(error_msg, "GetItem")
-    if path_param_key == "clusterId":
-        mock_emr.describe_cluster.side_effect = ClientError(error_msg, "DescribeCluster")
+    # if path_param_key == "clusterId":
+    #     mock_emr.describe_cluster.side_effect = ClientError(error_msg, "DescribeCluster")
```

Comment:
Do we need to keep these commented lines?

Reply:
ah yeah I missed that during testing, will remove.